supabase/functions/.env.example

```diff
@@ -2,27 +2,27 @@
 S3_ENDPOINT=localhost:54321/storage/v1/s3
 S3_REGION=local
 S3_SSL=false
-S3_ACCESS_KEY_ID=625729a08b95bf1b7ff351a663f3a23c
-S3_SECRET_ACCESS_KEY=850181e4652dd023b7a98c58ae0d2d34bd487ee0cc3254aed6eda37307425907
-S3_BUCKET=capgo
+S3_ACCESS_KEY_ID=***
+S3_SECRET_ACCESS_KEY=***
+S3_BUCKET=test-bucket
```

Comment:
This was the default value of any given Supabase local project, maybe we should leave them or explian in the doc that is the one you get from supabase start 

<img width="1384" height="270" alt="Image" src="https://github.com/user-attachments/assets/e15475ed-20cd-41ee-b6d6-40bff375e07f" />